pmd-java/src/main/java/net/sourceforge/pmd/lang/java/metrics/rule/DataClassRule.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.metrics.rule;
+
+import net.sourceforge.pmd.lang.java.ast.ASTAnyTypeDeclaration;
+import net.sourceforge.pmd.lang.java.metrics.JavaMetrics;
+import net.sourceforge.pmd.lang.java.metrics.api.JavaClassMetricKey;
+import net.sourceforge.pmd.lang.java.rule.AbstractJavaMetricsRule;
+
+/**
+ * @author Clément Fournier
+ * @since 6.0.0
+ */
+public class DataClassRule extends AbstractJavaMetricsRule {
+
+    // probably not worth using properties
```

Comment:
We could enhance the rule later, to make it configurable - if we find the need to do so.
But for now: Keep it simple :)